### PR TITLE
fix nan sec

### PR DESCRIPTION
**Pull request by APshenkin**
Don't display (NaN sec) for skipped steps in tests

### CODE REVIEW

File: lib/output.js [hunk 1]
```diff
@@ -94,11 +94,15 @@ module.exports = {
 
     if (outputLevel === 2) {
       newline = true;
-      process.stdout.write(`${styles.debug(" (" + time + ")\n")}`);
+      if (time !== 'NaN sec') {
```
Comment (DavertMik) [inline lib/output.js]: this check should be for integer values and not for strings...
please move this few lines upper where the `time` value is calculated

**Comment by DavertMik** [comment]
I think this is incorrect. Reproduced in Chrome console:

```
step = { startTime: 3, endTime: 5 }
{startTime: 3, endTime: 5}

 time = (step.endTime - step.startTime) / 1000 + " sec"
"0.002 sec"

isNaN(time)
true
```
isNaN check should happen before it was transformed to string

**Comment by APshenkin** [comment]
@DavertMik 
You're right! My bad. Now it's fixed

**Comment by DavertMik** [comment]
Ok, I will fix that manually

**Pull request merged**